.github/workflows/main.yml

```diff
@@ -45,17 +45,12 @@ jobs:
       run: make test-containers-test "CEPH_VERSION=${{ matrix.ceph_version }}" "RESULTS_DIR=$PWD/_results"
     - name: Clean up test containers
       run: make test-containers-clean "CEPH_VERSION=${{ matrix.ceph_version }}"
-    - name: Archive coverage results
-      uses: actions/upload-artifact@v2
+    - name: Archive test results
+      uses: actions/upload-artifact@v3
       with:
-        name: "go-ceph-coverage-${{ matrix.ceph_version }}"
-        path: "_results/coverage/go-ceph.html"
-    - name: Archive implements results
-      if: "matrix.ceph_version == 'pacific'"
-      uses: actions/upload-artifact@v2
-      with:
-        name: "go-ceph-implements-${{ matrix.ceph_version }}"
-        path: "_results/implements.*"
+        name: "go-ceph-results-${{ matrix.ceph_version }}"
+        path: |
+          _results/
```

Comment:
`path: _results/` also works, right?

Reply:
It should. The newer version can support multiple paths and exclusions and so the docs use the "|" line continuation syntax in some places. I changed it to partly serve as a reminder those options now exist.

Reply:
Ok.